src/mlpack/tests/main_tests/pca_test.cpp

```diff
@@ -18,21 +18,13 @@ static const std::string testName = "PrincipalComponentAnalysis";
 
 using namespace mlpack;
 
-// Utility function to set a parameter and mark it as passed, using copy
-// semantics.
-template<typename T>
-void SetInputParam(const std::string& name, const T& value)
-{
-  CLI::GetParam<T>(name) = value;
-  CLI::SetPassed(name);
-}
-
-// Utility function to set a parameter and mark it as passed, using move
-// semantics.
+// Utility function to set a parameter and mark it as passed,
+// using copy semantics for lvalues and move semantics for rvalues
```

Comment:
Nice, this is a good improvement. :+1:

Since this will almost certainly be useful in many places, maybe it would be useful to split this into a separate header file?